lib/services.js

```diff
@@ -12,6 +12,26 @@ ejs.filters.q = function(obj) {
   return JSON.stringify(obj, null, 2);
 };
 
+ejs.filters.getPropertyOfFirstEndpoint = function(remoteObj, item) {
+  return getEndPointsBackwardsCompatility(remoteObj, item);
+};
+
+// this method is added to support both flavors of the getEndPoints method
+// in strong-remoting 2.x methods `getHttpMethod()` and `getFullPath()` were used
+// in strong-remoting 3.x we have deprecated those methods in favor of getEndPoints()
+function getEndPointsBackwardsCompatility(remoteObj, item) {
+  // strong-remoting 3.x
+  if (typeof remoteObj.getEndpoints == 'function') {
+    return remoteObj.getEndpoints()[0][item];
+  } else { // strong-remoting 2.x
+    if (item == 'verb') {
+      return remoteObj.getHttpMethod();
+    } else if (item == 'fullPath') {
+      return remoteObj.getFullPath();
+    }
```

Comment:
What if `item` is neither `verb` nor `fullPath`? Should we throw an error instead of returning `undefined`?